pyproject.toml

```diff
@@ -64,7 +66,6 @@ GitPython = "~3.1.13"
 # GraphQL support
 graphene-django = "~2.15.0"
 # WSGI HTTP server
```

Comment:
Go ahead and remove this comment as well please. :-)

Reply:
Oh yeah `poetry remove gunicorn` left the comment.